<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dtkConceptDefinitionPage</class>
 <widget class="QWizardPage" name="dtkConceptDefinitionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;Concept Definition:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Select the ancestor(s) of your class. &lt;/p&gt;&lt;p&gt;Additionally if this concept is designed to be implemented in plugins, checfk the &amp;quot;Plugin system&amp;quot; box.  &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Inherits:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="m_runnable">
        <property name="text">
         <string>QRunnable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_qObject">
        <property name="text">
         <string>QObject</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_custom">
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_customName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_pluginSystem">
     <property name="text">
      <string>Plugin System</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
